host/xtest/regression_1000.c

```diff
@@ -2125,13 +2126,18 @@ static void xtest_tee_test_1027(ADBG_Case_t *c)
 
 out:
 	TEEC_CloseSession(&session);
+#else /*!OPENSSL_FOUND*/
+	UNUSED(c);
```

Comment:
I suggest adding a comment `/* xtest_uuid_v5() depends on OpenSSL */` because it's not totally obvious which part of the test depends on it. And same below.

Reply:
Will add.